src/azure-cli-core/azure/cli/core/commands/progress.py

```diff
@@ -0,0 +1,181 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+import sys
+import humanfriendly
+from enum import Enum
+
+BAR_LEN = 100
+
+
+class ProgressType(Enum):  # pylint: disable=too-few-public-methods
+    """ the types of progress """
+    Determinate = 0
+    Indeterminate = 1
+    Both = 2
```

Comment:
What is Both mean? Perhaps a more descriptive name or at least a comment to help explain?